jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -829,7 +827,7 @@ void jbmc_parse_optionst::process_goto_function(
     function.get_function_id(),
     goto_function,
     symbol_table,
-    get_message_handler());
+    log.get_message_handler());
```

Comment:
Nit pick: I think this change is in the wrong commit as it is a fix-up for the previous change.